src/Form/BasicSetup.php

```diff
@@ -134,15 +133,15 @@ public function buildForm(array $form, FormStateInterface $form_state, User $use
           $context['setup_context'] = ['plugin_definition' => $plugin_definition];
           $form['#title'] = t('TFA setup - Trusted browsers');
           $setup_plugin = new TfaTrustedBrowserSetup($context);
-          $tfa_setup = new TfaSetup($setup_plugin, $context);
+          $tfa_setup = new TfaSetup($setup_plugin);
```

Comment:
Those are for other plugins I havent tackled those yet.
If I removed it the module would break.